client/remoterepo.c

```diff
@@ -271,18 +320,53 @@ TDNFCheckHash(
 uint32_t
 TDNFCheckRepoMDFileHashFromMetalink(
     char *pszFile,
-    TDNF_METALINK_FILE *ml_file
+    TDNF_METALINK_CTX *ml_ctx
     )
 {
     uint32_t dwError = 0;
+    TDNF_METALINK_HASH_LIST *hashList = NULL;
+    TDNF_METALINK_HASH *hashInfo = NULL;
+    unsigned char digest[EVP_MAX_MD_SIZE];
+    int hashType = -1;
+
     if(IsNullOrEmptyString(pszFile) ||
-       !ml_file)
+       !ml_ctx)
     {
         dwError = ERROR_TDNF_INVALID_PARAMETER;
         BAIL_ON_TDNF_ERROR(dwError);
     }
 
-    dwError = TDNFCheckHash(pszFile, ml_file->digest, ml_file->type);
+    hashList = ml_ctx->hashes;
+    while(hashList)
+    {
```

Comment:
Something is wrong with indenting in this block. Do not use tabs, just spaces, 4 spaces for each level.

Reply:
Done